src/summary/history/patient-notes/notes-card-style.css

```diff
@@ -0,0 +1,140 @@
+.moreNotes {
+  display: inline-flex;
+  align-items: center;
+  border: none;
+  padding: 0.625rem 1.625rem 0.625rem 1rem;
+  background-color: white;
+  color: var(--omrs-color-ink-medium-contrast);
+  cursor: pointer;
+  text-decoration: none;
+}
+
+.tableData {
+  font-size: 1.0625rem;
+  text-align: center;
+}
+
+:global(.omrs-breakpoint-lt-tablet) .tableDates {
+  width: 70%;
+}
+
+table.table .tableDate {
+  font-family: "Work Sans";
+  font-weight: 500;
+}
+
+.location {
+  color: var(--omrs-color-ink-medium-contrast);
+  font-family: "Work Sans";
+}
+omrs-icon {
+  width: 1.5rem;
+  height: 0.9rem;
+  margin: 0rem;
+}
+.table {
+  width: 100%;
+}
+
+.tableRow {
+  border-bottom: 1px solid var(--omrs-color-bg-low-contrast);
+}
+
+.tableHeader,
+.tableData,
+.tableDate {
+  padding: 0.625rem;
+}
+
+.tableHeader {
+  color: var(--omrs-color-ink-medium-contrast);
+  font-size: 0.9375rem;
+  text-transform: capitalize;
+  font-weight: normal;
+  letter-spacing: 0.0275rem;
+  font-family: "Work Sans";
+  text-align: left;
+}
+
+.tableData {
+  font-size: 1.0625rem;
+  text-align: center;
+}
+
+:global(.omrs-breakpoint-lt-tablet) .tableDates {
+  width: 70%;
+}
+.centerItems {
+  display: flex;
+  align-items: center;
+}
+.tableDate {
+  font-family: "Work Sans";
+}
+
+table.table {
+  font-family: "Work Sans";
+  font-weight: 500;
+}
+
+.location {
+  color: var(--omrs-color-ink-medium-contrast);
+  font-family: "Work Sans";
+}
+@media only screen and (max-width: 760px),
+  (min-device-width: 490px) and (max-device-width: 1024px) {
+  /* Force table to not be like tables anymore */
+  table,
+  thead,
+  tbody,
+  th,
+  td,
+  tr {
+    display: block;
+  }
+  .tableDate {
+    font-family: "Work Sans";
+    font-weight: 500;
+  }
+  .moreNotes {
+    display: inline-flex;
+    align-items: center;
+    border: none;
+    padding: 0.625rem 1.625rem 0.625rem 1rem;
+    background-color: white;
+    color: var(--omrs-color-ink-medium-contrast);
+    cursor: pointer;
+    text-decoration: none;
+  }
+
+  /* Hide table headers (but not display: none;, for accessibility) */
```

Comment:
Can you explain why `display: none;` would be bad for accessibility here? Do we want it to be read to screen reader users, but not shown to visual users?